extras/travis_build.sh

```diff
@@ -282,8 +280,6 @@ echo "MSGBLD0270: Running make install."
 make install
 echo "MSGBLD0280: Make install completed."
 
-source $NEST_RESULT/bin/nest_vars.sh
-
```

Comment:
Instead of removing this line, I would prefer if
* this line were moved to around line 198 and 208 of the toplevel `CMakeLists.txt`
* the setting of `NEST_PATH` and `NEST_PYTHONPATH` were removed from around there
* the explicit setting of `PYTHONPATH` and `PATH` in line 352-356 of `testsuite/do_tests.sh` were removed.

In general, the two path variables should never be set explicitly, but `nest_vars.sh` should be sourced instead, in order to be consistent everywhere.

Reply:
The new solution is even better!